Fixed "missing social links Issue #235"

simsausaurabh opened the pull request:
Fixes #235 

***Added github icon at top menu bar and to the bottom***

![screenshot from 2017-12-21 19-26-31](https://user-images.githubusercontent.com/16608864/34258716-372523bc-e685-11e7-834c-a0d0dc30ff04.png)

![screenshot from 2017-12-21 19-26-38](https://user-images.githubusercontent.com/16608864/34258732-53b38474-e685-11e7-9934-b92eabfbe833.png)

vaibhavsingh97 commented on index.html:
```diff
@@ -26,6 +26,7 @@
         <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all"/>
         <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
+        <script src="https://use.fontawesome.com/40afd82a34.js"></script>
```
Use `https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css` instead of `https://use.fontawesome.com/40afd82a34.js`

anshumanv commented:
Please provide a test link.

0x48piraj commented:
@simsausaurabh If you're still working, you can add a **WIP**.

> It will be good if a mentor added the **WIP label**

simsausaurabh commented:
Test link: https://simsausaurabh.github.io/labs.fossasia.org/

0x48piraj commented:
Looks fine!
However, It's not perfectly align.
See the screenshot @simsausaurabh :+1: 
![image](https://user-images.githubusercontent.com/5800726/34274386-5c50a812-e6be-11e7-92a9-1bcf1689e30a.png)

simsausaurabh commented:
Yeah Working on it :smile:

simsausaurabh commented:
Although using same class for every icon it's not aligning properly. But by mismatching or making only few of icons use `fa fa-<social-media> fa-lg` class, I got almost this accurate alignment.

![screenshot from 2017-12-22 03-00-53](https://user-images.githubusercontent.com/16608864/34275900-31a8604e-e6c5-11e7-8b7e-0375555f54e6.png)

If it's okay then I shall make changes permanent. Or else if the older one is looking more aligned I shall leave it here. @0x48piraj @vaibhavsingh97 @anshumanv

0x48piraj commented:
![image](https://user-images.githubusercontent.com/5800726/34276295-be7cbc08-e6c6-11e7-86ef-25c0777fd660.png)

I think it's .. unaligned as before. :worried: 

Let mentors review this @simsausaurabh  :+1:

simsausaurabh commented:
No @0x48piraj I have added screenshot as a test output to show you how much accurate alignment I achieved. If it's not more accurate as the one in this PR then I shall leave it or else I shall make these testing changes permanent.

0x48piraj commented:
Why not, commit changes, give us a test link :smile:

simsausaurabh commented:
These are two perfectly aligned, please see which one is more better:  
***Square shaped***
https://simsausaurabh.github.io/square/

***Round shaped***
https://simsausaurabh.github.io/round/

@0x48piraj @vaibhavsingh97

vaibhavsingh97 commented:
Round shape looks good 👍

mariobehling commented:
Please keep it simple and try to be savy with the time of the volunteer mentors by following best practices as you learn more how to contribute in our projects. Thank you!

Check it out here: https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/

simsausaurabh commented:
Thank you @mariobehling for review.
I have removed the unwanted changes and added github icon. Please review @mariobehling @vaibhavsingh97 @0x48piraj .

mariobehling commented:
Thanks.
